configure.ac

```diff
@@ -861,7 +861,8 @@ if test "x$enable_static_libevent" = "xyes"; then
    if test "x$tor_cv_library_libevent_dir" = "x(system)"; then
      AC_MSG_ERROR("You must specify an explicit --with-libevent-dir=x option when using --enable-static-libevent")
    else
-     TOR_LIBEVENT_LIBS="$TOR_LIBDIR_libevent/libevent.a $STATIC_LIBEVENT_FLAGS"
+     dnl If libevent was _not_ installed, the library is in .libs/.
+     TOR_LIBEVENT_LIBS="$TOR_LIBDIR_libevent/libevent.a $TOR_LIBDIR_libevent/.libs/libevent.a $STATIC_LIBEVENT_FLAGS"
```

Comment:
Does this work? I think this would make us expect different libraries in the two different locations.  What happens if both libraries are present? Maybe we should check which one we have.